docs/modules/extensions/pages/processors/inline-macro-processor.adoc

```diff
@@ -3,6 +3,15 @@
 
 InlineMacroProcessor is a SyntaxProcessor
 
+Note:
+Currently, uses of inline macros are recognized and processed during the conversion phase of a parsed block.
+Furthermore, uses are recognized by means of a regular expression.
+One of the goals of many participating the in Asciidoc language standardization effort is to sufficiently define the language so that inline macro uses can be recognized and processed during the parse phase, so the inline Nodes returned from processing form part of the DOM or parse tree.
```

Comment:
```suggestion
One of the goals of many participating in the AsciiDoc language standardization effort is to sufficiently define the language so that inline macro uses can be recognized and processed during the parse phase, so the inline Nodes returned from processing form part of the DOM or parse tree.
```

I think this sentence is a bit long. Maybe we should use a dot and not a comma?
I also think that we should expand the acronym DOM and probably state that we are not talking about HTML DOM.

Reply:
Yes. Rewritten a bit.